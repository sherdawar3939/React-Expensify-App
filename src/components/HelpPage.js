import React from 'react';
const ExpenseHelpPage = ()=>(
    <div>
     this is From Help expense page
    </div>
)
export default ExpenseHelpPage