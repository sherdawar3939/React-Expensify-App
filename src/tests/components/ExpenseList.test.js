import React from 'react';
import { shallow  } from 'enzyme';
import { ExpenseList } from '../../components/ExpenseList';
import toJson from 'enzyme-to-json';
import expenses from '../fixtures/expenses';

test('should render expense list expenses', ()=>{
    const wrapper = shallow(<ExpenseList expenses={expenses}/>);
    expect(toJson(wrapper)).toMatchSnapshot()
})


test('should render expense list with empty array', ()=>{
    const wrapper = shallow(<ExpenseList expenses={[]}/>);
    expect(toJson(wrapper)).toMatchSnapshot()
})