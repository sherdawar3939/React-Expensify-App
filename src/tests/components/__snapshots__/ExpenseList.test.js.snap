// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list expenses 1`] = `
<div>
  <ExpenseListItem
    amount={123}
    createdAt={0}
    description="gum"
    id="1"
    key="1"
    note=""
  />
  <ExpenseListItem
    amount={12309}
    createdAt={-345600000}
    description="rent"
    id="2"
    key="2"
    note=""
  />
  <ExpenseListItem
    amount={12322}
    createdAt={345600000}
    description="credit card"
    id="3"
    key="3"
    note=""
  />
</div>
`;

exports[`should render expense list with empty array 1`] = `
<div>
  <p>
    No Expenses
  </p>
</div>
`;
